qa/tasks/radosgw_admin.py

```diff
@@ -257,6 +263,36 @@ def remove_newlines(string):
         remove_newlines(raw_acl)
     )
 
+def cleanup(ctx, client, omit_sudo=False, omit_tdir=False):
+    # remove objects and buckets
+    (err, out) = rgwadmin(ctx, client, ['bucket', 'list'], check_status=True, omit_sudo=omit_sudo, omit_tdir=omit_tdir)
```

Comment:
to avoid this proliferation of `omit_sudo=omit_sudo, omit_tdir=omit_tdir` everywhere, what do you think about making the `rgwadmin` function a global variable that remembers which configuration we're in? if we add new tests that forget these flags, they would pass in teuthology but would fail under local runner

something like this?
```python
import util.rgw
rgwadmin = util.rgw.rgwadmin

def main():
    def rgwadmin_local(*args, **kwargs):
        util.rgw.rgwadmin(*args, **kwargs, omit_sudo=True, omit_tdir=True)
    rgwadmin = rgwadmin_local

    rgwadmin('foo', {'bar':'baz'}, kw='value') # also passes omit_sudo and omit_tdir

if __name__ == '__main__':
        main()
```